test/e2e/framework/gateways.go

```diff
@@ -198,3 +198,33 @@ func (f *Framework) GatewayCleanup() {
 		}
 	}
 }
+
+// Perform a gateway failover.
+// The failover for the real environment will crash the gateway node.
+// THe failover for the kind environment will set the "false" label on the gw node.
+func (f *Framework) DoFailover(cluster ClusterIndex, gwNode, gwPod string) {
+	provider := DetectProvider(cluster, gwNode)
+
+	if provider == "kind" {
+		f.SaveGatewayNode(cluster, gwNode)
+		f.SetGatewayLabelOnNode(cluster, gwNode, false)
+	} else {
+		cmd := []string{"sh", "-c", "echo 1 > /proc/sys/kernel/sysrq && echo b > /proc/sysrq-trigger"}
```

Comment:
After running this command, is the node getting restarted?

Reply:
Yes, the node is crashed and restarted.
After boots up, it continue to work as expected.